ddht/v5_1/alexandria/abc.py

```diff
@@ -239,6 +285,8 @@ class AlexandriaNetworkAPI(ServiceAPI, TalkProtocolAPI):
     client: AlexandriaClientAPI
     routing_table: RoutingTableAPI
 
+    advertisement_db: AdvertisementDatabaseAPI
```

Comment:
was expecting to see the `Storage` pattern, like `ContentStorageAPI`, so that an in-memory option could also be used. does that not work here for some reason?

Reply:
The "in memory" version is just one that uses `sqlite3.connect(":memory:")` which creates an in-memory sqlite3 database.